instrumentors/tox.ini

```diff
@@ -0,0 +1,41 @@
+[tox]
+skipsdist = True
+skip_missing_interpreters = True
+envlist =
+; Environments are organized by individual package, allowing
+; for specifying supported Python versions per package.
+
+  py3{4,5,6,7,8}-test-flask
+  pypy3-test-flask
+
+  py3{4,5,6,7,8}-test-django
+  pypy3-test-django
+
+[testenv]
+deps =
+  -c dev-requirements.txt
+  test: pytest
+; FIXME: add coverage testing
+; FIXME: add mypy testing
+; FIXME: add lint testing
+; FIXME: add travis configuration
+
+changedir =
+  flask: flask/tests
+  django: django/tests
+
+commands_pre =
+; Install without -e to test the actual installation
+  python -m pip install -U pip setuptools wheel
+;  test: pip install {toxinidir}/flask
+  test: pip install {toxinidir}/utils/opentelemetry-auto-instrumentation
+  test: pip install {toxinidir}/utils/opentelemetry-ext-testutil
+
+  flask: pip install {toxinidir}/flask
```

Comment:
We don't have those packages now. Once they are present the path should be `instrumentors/`, right?

Reply:
Yes, but I don't think we will at all, I'll remove them from the tox.ini file.